common/strings.h

```diff
@@ -44,9 +44,10 @@
 #include <tuple>
 #include <type_traits>
 
+#include <fmt/format.h>
+
 #ifndef _WIN32
```

Comment:
Do we need this anymore if there’s nothing inside it?

Reply:
It’s a ifndef with an else

Reply:
:+1: